### ISSUE TITLE
Address display formating

### ISSUE THREAD

**Issue by pythcoiner** [wizardsardine/liana#837]
Should we change the address display format to something more readable like it's done on specter?
![image](https://github.com/wizardsardine/liana/assets/124568858/d5346778-6877-4cd3-8739-47458299eafa)

**Comment by darosior** [comment]
Not sure. It's important on a device where you'd have to verify the address, such as on your hardware signing device. But verifying the address on the laptop doesn't get you very far IMO.

**Comment by pythcoiner** [comment]
my mean to compare between the one displayed on the laptop and the signer

**Comment by darosior** [comment]
I don't think this achieves much to be honest. It's good to be able to make the signer display the address, but if you assume the laptop is compromised (the only threat model which makes sense for using a hardware signer) then verifying it on the screen doesn't mean it's what will end up being shared.

**Comment by pythcoiner** [comment]
related to #377
